docs/source/tt-metalium/tools/tracy_profiler.rst

```diff
@@ -133,6 +133,34 @@ Similar to the pytest setup, calling the parent script with ``-p`` option will p
 **Note**, it is recommended to sandwich the function call between the enable and disable calls rather than having them as first and last calls in the function being profiled.
 This is because ``settrace`` and ``setprofile`` trigger on more relevant events when the setup is done previous to the function call.
 
+Signposts in Python Code
```

Comment:
@nsmithtt Wanted to make sure this is inline with what you had in mind as well.

Reply:
This is exactly what I had in mind, thank you!